ui/components/multichain/create-account/create-account.tsx

```diff
@@ -91,6 +116,24 @@ export const CreateAccount: CreateAccountComponent = React.memo(
         trimmedAccountName || defaultAccountName,
         defaultAccountName,
       );
+      ///: BEGIN:ONLY_INCLUDE_IF(multi-srp)
+      const keyrings: {
+        accounts: InternalAccount[];
+        type: KeyringTypes;
+        metadata: { id: string; name: string };
+      }[] = useSelector(getMetaMaskKeyrings);
+      const hdKeyrings = keyrings.filter(
+        (keyring) => keyring.type === KeyringTypes.hd,
+      );
```

Comment:
Similar to my previous comment, having a `getMetaMaskHdKeyrings` selector could be handy here too.